lib/reporters/base.js

```diff
@@ -231,13 +235,16 @@ exports.list = function(failures) {
  * of tests passed / failed etc.
  *
  * @param {Runner} runner
+ * @param {Object} options runner optional options
  * @api public
  */
 
-function Base(runner) {
+function Base(runner, options) {
   var stats = this.stats = { suites: 0, tests: 0, passes: 0, pending: 0, failures: 0 };
   var failures = this.failures = [];
 
+  this.reporterOptions = options ? options.reporterOptions : null;
```

Comment:
Why not already call `openOutput` in the constructor? If you specify an `output` you probably want any logs coming from the reporter to go to that file, by default.

Also, not sure about the naming, `output`. Not very specific, and output can also mean 'resulting data'. It's always a file right? What about `outfile`?